controllers/announcements.go

```diff
@@ -0,0 +1,102 @@
+package controllers
+
+import (
+    "strconv"
+    "net/http"
+    "io/ioutil"
+    "encoding/json"
+    "gopkg.in/macaron.v1"
+)
+
+type Topic struct {
+    ID int `json:"id"`
+    Title string `json:"title"`
+    Slug string `json:"slug"`
+}
+
+type TopicList struct {
+    Topics []Topic `json:"topics"`
+}
+
+type Category struct {
+    TopicList TopicList `json:"topic_list"`
+}
+
+type Post struct {
+    Cooked string `json:"cooked"`
+}
+
+type PostStream struct {
+    Posts []Post `json:"posts"`
+}
+
+type TopicResponse struct {
+    PostStream PostStream `json:"post_stream"`
+}
+
+type AnnouncementView struct {
+    First Announcement `json:"first"`
+    Second Announcement `json:"second"`
+}
+
+type Announcement struct {
+    Title string `json:"title"`
+    Content string `json:"content"`
+    Link string `json:"link"`
+}
+
+func GetAnnouncements(ctx *macaron.Context) {
+    r, err := http.Get("https://forums.spongepowered.org/c/announcements.json?order=created")
+    body, err := ioutil.ReadAll(r.Body)
```

Comment:
No need to read + buffer everything first, there is a `json.Decoder` you can use directly on the body, see http://blog.golang.org/json-and-go#TOC_7.